CRM/Queue/DAO/Queue.php

```diff
@@ -145,21 +181,89 @@ public static function &fields() {
           ],
           'add' => '5.47',
         ],
-        'is_autorun' => [
-          'name' => 'is_autorun',
-          'type' => CRM_Utils_Type::T_BOOLEAN,
-          'title' => ts('Enable Autorun'),
-          'description' => ts('Should the standard background attempt to autorun tasks in this queue?'),
-          'where' => 'civicrm_queue.is_autorun',
+        'runner' => [
```

Comment:
I guess it's not required but it feels like we could have a callback to get the list of registered options

Reply:
I was thinking about that too...

If sysadmins were using a GUI to define new queues, then it would be important to have that list. But I don't personally see the use-case for it right now. AFAICS, the use-cases are developer-driven ("Copy-paste 8-line snippet from dev-docs.")

The flip-side is that adding any `pseudoconstant` for `runner` would mean it becomes an affirmative obligation to register new values (otherwise validation will fail). So any custom runner requires 2x registrations. [Compare: Registering a queue with custom runner. Should it require 1x records (`Queue`) or 2x records (`Queue`+`OptionValue`)](https://gist.github.com/totten/db374657a1263024a04f0edaf3e295f7).

Reply:
@totten yeah - it does mean it becomes harder to add later - but I am ok with leaving for now

Reply:
note I don't really see having to do 2 registrations as a downside